### ISSUE TITLE
Markdown Highlighting Seems Off

### ISSUE THREAD

**Issue by benjamin-kirkbride** [Akuli/porcupine#1304]
Related to https://github.com/Akuli/porcupine/pull/1138 ?

![image](https://github.com/Akuli/porcupine/assets/3373830/59446d0c-5d4b-40d3-a4c1-039b982687f7)

Is this what we expect to see?

**Comment by rdbende** [comment]
Yes, currently it uses plain colors, not font styles, but there's a preview plugin in plans: #769

**Comment by Akuli** [comment]
I love your test file :D

**Comment by benjamin-kirkbride** [comment]
@Akuli yeah I started to make it then stole from you haha

**Comment by benjamin-kirkbride** [comment]
The thing that seems the most off is that the headings are not highlighted.

**Comment by benjamin-kirkbride** [comment]
This is how `vscode` looks by default (I believe) for comparison:
![image](https://github.com/Akuli/porcupine/assets/3373830/7f93f2d9-59bc-44d2-8b7f-46cb36445b45)
